plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/AbstractProductFormTemplate.php

```diff
@@ -55,4 +56,11 @@ public function add_group( array $block_config ): GroupInterface {
 		$block = new Group( $block_config, $this->get_root_template(), $this );
 		return $this->add_inner_block( $block );
 	}
+
+	/**
+	 * Add the default blocks.
+	 */
+	protected function add_default_blocks() {
```

Comment:
This is optional, but if we find that most templates share very similar blocks we can add them to this method in the template abstraction.

This way, templates can add or remove blocks specific to its own template.